include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -63,8 +63,11 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     ////////////////////////////////////////////////////////////
     /// \brief Open a sound file from the disk for reading
     ///
-    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC.
+    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC, MP3.
     /// The supported sample sizes for FLAC and WAV are 8, 16, 24 and 32 bit.
+    /// Because of minimp3_ex limitation, for MP3 files with big (>16kb) APEv2 tag,
```

Comment:
Empty line (only `///`) before, to create separate paragraph for the implementation details.